### PR TITLE
Improve light clipping and culling

### PR DESCRIPTION
**Pull request by EvgeniiG**
### Purpose of this PR
Improve light culling. The current implementation is not numerically stable:
https://fogbugz.unity3d.com/f/cases/1085873
Also clip box lights to the range:
https://fogbugz.unity3d.com/f/cases/1178780/

---
### Testing status
**Katana Tests**: 
---
### Comments to reviewers
Currently, area light culling is broken due to the frustum culling code at the C++ level.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs [hunk 1]
```diff
@@ -1540,63 +1540,59 @@ static Vector3 EvaluateAtmosphericAttenuation(Vector3 L, Vector3 positionWS)
...
 
-                lightVolumeData.lightPos = centerVS;
+                lightVolumeData.lightPos   = centerVS;
                 lightVolumeData.lightAxisX = xAxisVS;
                 lightVolumeData.lightAxisY = yAxisVS;
                 lightVolumeData.lightAxisZ = zAxisVS;
-                lightVolumeData.boxInnerDist = extents;
-                lightVolumeData.boxInvRange.Set(Mathf.Infinity, Mathf.Infinity, Mathf.Infinity);
-                lightVolumeData.featureFlags = (uint)LightFeatureFlags.Area;
+                lightVolumeData.boxInvRange.Set(1.0f / extents.x, 1.0f / extents.y, 1.0f / extents.z);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs]: is there a reasosn that we don't do this line anymore?
lightVolumeData.featureFlags = (uint)LightFeatureFlags.Area;

**Comment by sebastienlagarde** [comment]
Please add a changelog, thanks

**Comment by sebastienlagarde** [comment]
adding  @FrancescoC-unity  for awareness (for your PR about box bounds)

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs (hunk 1 above)
Comment (EvgeniiG) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs]: innerDist is 0 by default. Feature Flags is my mistake. I will add it back. Good catch!

**Comment by EvgeniiG** [comment]
I added feature flags again, but the PR is not updating. >.<

**Comment by FrancescoC-unity** [comment]
lgtm (and is not impacting my PR beyond a trivial merge conflict that'll be there @sebastienlagarde)

**Comment by sebastienlagarde** [comment]
@EvgeniiG  I don't see your commit that add back the feature flag. Guess it is not pushed. Maybe do a new PR if you have problem with this branch.

**Comment by sebastienlagarde** [comment]
+ please add changelog, thanks

**Comment by EvgeniiG** [comment]
My commits are not showing up: https://github.com/Unity-Technologies/ScriptableRenderPipeline/commits/HDRP/improve-area-light-culling
Will create another PR.

**Pull request closed without merging**